logos/logos-ui/src/app/features/model-error-report/model-error-report.scss

```
@@ -0,0 +1,1004 @@
+  display: inline-flex;
+  align-items: center;
+  gap: 6px;
+
+  margin-bottom: 16px;
+
+  font-size: var(--font-size-sm);
+  font-weight: var(--font-weight-medium);
+
+  color:
+    rgb(
+      var(--nav-item-text) /
```

Stylelint (17.14.0)

[error] Unexpected newline after "/" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)